### PR TITLE
Revert target framework version change to netcoreapp3.0 when building from VS

### PR DESCRIPTION
**Pull request by fadimounir**
Broken by https://github.com/dotnet/runtime/pull/445 (cc @ViktorHofer)

Using 3.0 for now enables us to build from VS, which is an important scenario for our team currently

@dotnet/crossgen-contrib

### CODE REVIEW

**Comment by ViktorHofer** [comment]
@fadimounir sorry for the break here. Can't you just install the 5.0 SDK instead?

**Comment by fadimounir** [comment]
@ViktorHofer I haven't tried installing 5.0. Does that work with VS today?

**Comment by ViktorHofer** [comment]
It should yes, but let me try it first.

**Comment by fadimounir** [comment]
> It should yes, but let me try it first.

I just tried it. It doesn't work

File: src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj
Comment (ViktorHofer) [inline src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj]: OK, I think the right fix would be to downgrade this project entirely. So to set TargetFramework to netcoreapp3.0 not just for inside VS as it was before. We can upgrade at a late rpoint. Same for crossgen2.

Comment (fadimounir) [reply src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj]: I actually think it was a good idea when you upgraded it to 5.0, since this is the shipping configuration.

The netcoreapp3.0 is just a temporary internal scenario used by our team, and we can remove it when dotnet5 starts working with VS.

**Review by trylek — APPROVED** [review]
I'm fine with the change but I would prefer to hear from Viktor that he's fine with whatever we're doing before merging in.

**Comment by ViktorHofer** [comment]
I will take a look early tomorrow. We can merge this in and if something points out I will let you know.

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**
